1 The Physical/Guess Point Level 1 (Stateless)/1 - Red-Green-Refactor Practice/1.1 - Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -4,9 +4,15 @@ function fizzbuzz(inputs: number[]) {
     if (currVal > 100) throw new Error("More that 100 is not allowed");
 
     let valueToAdd = currVal.toString();
-    if (currVal % 3 === 0) valueToAdd = "Fizz";
-    if (currVal % 5 === 0) valueToAdd = "Buzz";
-    if (currVal % 3 === 0 && currVal % 5 === 0) valueToAdd = "FizzBuzz";
+
+    const isMultipleOfThree = currVal % 3 === 0;
+    if (isMultipleOfThree) valueToAdd = "Fizz";
```

Comment:
I like that you refactored these lines to make their purpose clearer through naming